flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/FlinkStreamRuleSets.scala

```diff
@@ -246,7 +246,7 @@ object FlinkStreamRuleSets {
     PushFilterIntoLegacyTableSourceScanRule.INSTANCE,
```

Comment:
Why keep `PushFilterIntoTableSourceScanRule.INSTANCE`?

Reply:
I mean the line 245.

Reply:
Not sure what you mean by keep? Do you think it should be removed?